<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>541</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grd_main"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="menu_new"/>
    <addaction name="menu_openfile"/>
    <addaction name="menu_solve"/>
    <addaction name="menu_exit_2"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="menu_dark"/>
    <addaction name="menu_light"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menu_help_2"/>
    <addaction name="menu_about_2"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuTheme"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="menu_openfile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="menu_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="menu_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menu_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menu_dark">
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="menu_light">
   <property name="text">
    <string>Light Mode</string>
   </property>
  </action>
  <action name="menu_solve">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="menu_help_2">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="menu_about_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menu_exit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
